jekyll/_cci2/templates/template-overview.adoc

```diff
@@ -0,0 +1,153 @@
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= Page title
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A short page description goes here
+:icons: font
+:experimental:
+
+////
+Some notes on attributes
+
+:page-liquid: - ensures that all liquid tags are processed before rendering the content. More info here: https://github.com/asciidoctor/jekyll-asciidoc/blob/89b8f55f5312e4a0f1bca496bd9217d53d5b21dd/docs/modules/ROOT/pages/liquid.adoc
+
+:icons: font - enables the use of font awesome icons https://docs.asciidoctor.org/asciidoc/latest/macros/icons-font/
+
+:experimental: allows access to asciidoc macros, more info here: https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/
+
+////
+
+This template will help you create a new docs page to provide conceptual information about a feature or task in CircleCI. To use this template make a copy and place it in the `_cci2` directory. The filename should match the page title. Look at the source file for this page template link:https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/template/template-overview.adoc?plain=1[here]. There is additional detail in the comments.
+
+// If possible display the use cases in a bullet list:
+
+The opening paragraph should be used to succinctly explain to the reader what value the feature/task provides and what use cases it supports. Content templates help you to:
+
+* Develop new content quickly
+* Ensure your page conforms to the style guide
+
+[#introduction]
+== Introduction
+
+Use the introduction to explain more about the feature/task, what it is for, why it is useful, and the wider context. Try to answer the question “What’s in it for me?” for the reader.
+
+You might want to include an image or diagram here to help illustrate the concept. The docs team can work with you to help create helpful image to support explanitory information:
+
+image::{{site.baseurl}}/assets/img/docs/arch.png[Provide some alt text for images here]
+
+[#quickstart]
+== Quickstart
+
+Some readers will want to read through a full overview page to get a deep understanding of how a feature works, the full set of options available to them, any gotchas or limitations that might apply to them, and to find troubleshooting information. Some readers will want to get started with the feature or task right away and then return to the docs as needed. For the latter it is a good idea to provide a quickstart section to help get the reader up and running with the basics in just a few steps:
+
+// The following will render as a numbered list
+
+. Provide some steps here
+. To get started with this feature
+. Inlcude a step for every action
+. Try not to skip steps or assume readers will know which button to click etc.
+
+Alternatively, point readers to a how-to guide or tutorial in this quickstart section.
```

Comment:
I'm actually thinking we put a preference on linking to an existing how-to / tutorial, and _then_ as an alternative outline a quickstart if the other docs aren't available. How does that sound?

Reply:
Yep sounds good!